auau/augmenter.py

```diff
@@ -98,15 +98,25 @@ def pitch_scale(self, signal, sr, num_semitones=2):
         augmented_signal = FAudio.pitch_shift(signal, sr, num_semitones)
         return self._tensor_on_device(augmented_signal)
 
-    def random_gain(self, signal, min_gain=1, max_gain=1.2):
-        """Random gain, processing on device
+    def gain(self, signal, gain):
```

Comment:
I'm guessing this might be of some use to somebody. Instead of introducing a fixed (random) gain throughout the entire signal, it may be useful to (1) allow the user to select the gain they want) and (2) allow the gain to be a function of time (i.e. gain can also be a tensor that multiplies the original signal, element-wise)